test/python/circuit/test_gate_power.py

```diff
@@ -32,15 +61,13 @@ def test_sgate_int(self, n):
         """Test Sgate.power(n) method with n as integer."""
         result = SGate().power(n)
 
-        self.assertEqual(result.label, "s^%s" % n)
-        self.assertIsInstance(result, UnitaryGate)
```

Comment:
Would it be better if we instead check for the PhaseGate type now instead of just removing the check?

Reply:
Done.